stack.yaml

```diff
@@ -1 +1 @@
-resolver: lts-17.4
+resolver: lts-18.10
```

Comment:
Note that CI is ignoring this configuration. Instead it's using explicitly installed ghc + --system-ghc flag for stack.
But it's ok to update this anywa.

Reply:
I updated it because of my local workflow. 17.4 comes with ghc version that I would need to compile from source (takes ages).